### ISSUE TITLE
Permision denied while running Tobira

### ISSUE THREAD

**Issue by mliradelc** [elan-ev/tobira#639]
Hi, I have a very strange error with Tobira, every time I try to execute it, gave me this message:

```
$ ./tobira check


▶▶▶ Error: failed to load config: cannot proceed with `check` command

Caused by:
 ‣ Permission denied (os error 13)
```

I'm trying with the version 1.5 of tobira, about the rights I'm doing the exact steps using Ansible example from here https://elan-ev.github.io/tobira/setup/ansible-example#tobira

Just checked:

- Name and group | Tobira group and user are correct
- The rights from the folder and subfolder and files. | All correct.

The only thing that I have not checked been how Tobira was built, but I'm using the official release from GitHub.

If someone have any other idea would be great.

**Comment by JulianKniephoff** [comment]
> ```
> $ ./tobira check
> ```

Whereever you executed this, can you give me the output of `pwd`, `ls -l` and `id`?

**Comment by mliradelc** [comment]
Hi Julian, here are the parameters you asked:

```
[tobira@pc tobira]$ pwd
/opt/tobira
[tobira@pc tobira]$ ls -l
total 25668
-rw-r--r-- 1 tobira tobira    14827 Dec 21 11:31 config.toml
-rw-r--r-- 1 tobira tobira      522 Dec 21 11:31 favicon.svg
-rw-r--r-- 1 tobira tobira    17687 Dec 21 11:31 logo-large.svg
-rw-r--r-- 1 tobira tobira     1262 Dec 21 11:31 logo-small.svg
-rw-r--r-- 1 tobira tobira      241 Dec 21 11:31 private-key.pem
-rwxr-xr-x 1 tobira tobira 26233384 Dec 21 11:31 tobira
[tobira@atlan tobira]$ id
uid=7968(tobira) gid=7968(tobira) groups=7968(tobira)
[tobira@pc tobira]$ whoami
tobira
```

**Comment by JulianKniephoff** [comment]
OK, that looks right at a first glance for the most part, **except** for the prompt changing for the `id` command. What happened there? :thinking:

**Comment by JulianKniephoff** [comment]
Unrelated: The first thought that pops into my mind when I see permission errors despite the permissions looking right is: SELinux. Maybe that helps you? :shrug:

**Comment by mliradelc** [comment]
I found the issue, the problem is if you set a logfile outside the user tobira.

I set `/var/log/tobira.log` as the logfile.

How I found it? I asked ChatGPT about the problem, the program toldme about the program `strace`, that gives you all the system calls from the command you executes.

Then, you run:
`strace ./myprogram 2>&1 | grep "Permission denied"`

And here is the issue:
```
openat(AT_FDCWD, "/var/log/tobira.log", O_WRONLY|O_CREAT|O_APPEND|O_CLOEXEC, 0666) = -1 EACCES (Permission denied)
write(2, "Permission denied", 17Permission denied)       = 17
```

is ti possible to add a stacktrace to Tobira?
